yellowbrick/features/importances.py

```diff
@@ -129,23 +130,22 @@ def fit(self, X, y=None, **kwargs):
         # Get the feature importances from the model
         self.feature_importances_ = self._find_importances_param()
 
+        # Get the classes from the model
+        self.classes_ = self._find_classes_param(y)
```

Comment:
So this is a bit tricky, this parameter is only used if `self.stack == True` and in the case of regression, this is simply the unique values in the regressor (which can be quite a lot of values) and does not refer to classes. I appreciate that this must come before Line 139 because `_find_classes_param` will set `stack=False`. 

Instead, I propose that you use `yellowbrick.utils.is_classifier` to check if the model is a classifier, then fetch the classes from the estimator, otherwise raise an exception that with stack=True only multi-output classifiers are appropriate. It would be best if `self.classes_` was only stored for classification targets. Would you mind also adding documentation for the `self.classes_` attribute in the docstring?

Note that alternatively you could use [`sklearn.utils.multiclass.type_of_target`](http://scikit-learn.org/dev/modules/generated/sklearn.utils.multiclass.type_of_target.html) to check the type of the target; if the result is in `{"binary", "multiclass", "mutliclass-multioutput"}` then it is a classification target, if it is in `{"continuous", "continuous-multioutput"}` then it is a regressor; but this is more important for #508 and #524 than it is for this.

Reply:
Thanks for pointing this out.

I will do a `yellowbrick.utils.is_classifier` check before line 139 and add `self.classes_` to docstring.